src/main/dialogs/captureToolDialog.js

```diff
@@ -0,0 +1,294 @@
+
+require("../../../node_modules/gpii-universal/gpii/node_modules/solutionsRegistry/src/js/SolutionsUtils.js");
+
+fluid.defaults("gpii.app.captureToolUtils", {
+    gradeNames: ["fluid.component"],
+    events: {
+        onCollectDiagnostics: null
+    },
+    listeners: {
+        onCollectDiagnostics: [{
+            funcName: "gpii.flowManager.getSolutionsPromise",
+            args: [ "{flowManager}.solutionsRegistryDataSource", null]
+        }, {
+            funcName: "gpii.app.captureToolUtils.collectPayload",
+            args: ["{arguments}.0", "{arguments}.1", "solutions"]
+        }, {
+            func: "{flowManager}.capture.getInstalledSolutions"
+        }, {
+            funcName: "gpii.app.captureToolUtils.collectPayload",
+            args: ["{arguments}.0", "{arguments}.1", "installedSolutions"]
+        }, {
+            func: "{flowManager}.capture.getSystemSettingsCapture"
+        }, {
+            funcName: "gpii.app.captureToolUtils.collectPayload",
+            args: ["{arguments}.0", "{arguments}.1", "settingsCapture"]
+        }]
+    }
+});
+
+gpii.app.captureToolUtils.collectPayload = function (value, options, keyName) {
+    options[keyName] = value;
+    return options;
+};
+
+fluid.defaults("gpii.app.captureTool", {
+    gradeNames: [
+        "gpii.app.dialog",
+        "gpii.app.centeredDialog",
+        "fluid.modelComponent"
+    ],
+    model: {
+        isKeyedIn: false,
+        keyedInUserToken: null,
+        preferences: {}
+    },
+    /*
+     * Raw options to be passed to the Electron `BrowserWindow` that is created.
+     */
+    config: {
+        destroyOnClose: false,
+        fileSuffixPath: "captureTool/index.html",
+
+        params: {
+            theme: "{that}.model.theme"
+        },
+
+        attrs: {
+            icon: {
+                expander: {
+                    funcName: "fluid.module.resolvePath",
+                    args: ["%gpii-app/src/icons/Morphic-Desktop-Icon.ico"]
+                }
+            },
+            alwaysOnTop: false,
+            frame: true,
+            resizable: true,
+            transparent: false, // needs to be false to enable resizing and maximizing
+            movable: true,
+            minimizable: true,
+            maximizable: true,
+            autoHideMenuBar: true,
+            // titleBarStyle: "hidden"
+            titleBarStyle: "default"
+        }
+    },
+    invokers: {
+        updateRenderModel: {
+            funcName: "gpii.app.captureTool.updateRenderModel",
+            args: ["{that}", "{arguments}.0", "{arguments}.1"]
+        },
+        testPspChannel: {
+            funcName: "gpii.app.captureTool.testPspChannel",
+            args: ["{that}", "{pspChannel}", "{flowManager}", "{arguments}.0"]
+        },
+        generateDiagnostics: {
+            funcName: "gpii.app.captureTool.generateDiagnostics",
+            args: ["{that}", "{flowManager}"]
+        }
+    },
+    listeners: {
+        onCreate: [
```

Comment:
Similarly, namespace and prioritise these

Reply:
done